src/routers/users.js

```diff
@@ -23,4 +24,12 @@ usersRouter.post(
 usersRouter.patch('/block/:id', usersController.blockUser);
 usersRouter.patch('/delete/:id', usersController.deleteUser);
 
+usersRouter.post(
+  '/cart/:id',
```

Comment:
We must get id from req.session.passport.user object instead of passing it as req param, because in current case we can add products to any user cart

Reply:
@VasylLymych, in this case, should we create a new user in MongoDB when someone logs in through Google?

Google ID's are 21 chars long, numbers only. They don't match MongoDB id object (BSON objectId).

Reply:
Probably yes

Reply:
should this be applied to both getProducts and deleteProducts as well ?

Reply:
Yes